frontend/src/app/tasks/hooks/useTaskTypes.ts

```diff
@@ -0,0 +1,36 @@
+import { useQuery } from '@tanstack/react-query'
+import { useAuth } from 'react-oidc-context'
+
+type Task = {
```

Comment:
This should probably be named `TaskType` to avoid confusion with the tasks later on.

Reply:
oops, I started with useTask and forgot to change this type to TaskType, tnx